src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java

```diff
@@ -87,6 +90,11 @@ public AboutDialogContents(ProductInfo info, ProductEditionInfo editionInfo)
             " is provided by RStudio, PBC for testing purposes only and is not an officially supported release."
          );
       }
+      else
+      {
+         preReleaseRibbon.setVisible(false);
```

Comment:
Is the pre-release ribbon visible by default?

Reply:
Yes, it just gets turned off in release builds. There's no visual difference either way, but if you think it'd make more sense I could have it hidden by default and then only shown in pre-release builds.

Reply:
Seems fine to me! I just couldn't see where it gets enabled, so I wanted to confirm my assumption.